api/envoy/extensions/upstreams/http/generic/v3/generic_connection_pool.proto

```diff
@@ -14,5 +14,5 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // A connection pool which forwards downstream HTTP as TCP or HTTP to upstream,
 // based on CONNECT configuration.
 // [#extension: envoy.upstreams.http.generic]
-message GenericConnectionPoolProto {
+message GenericConnectionPool {
```

Comment:
This is a breaking API change, see https://github.com/envoyproxy/envoy/blob/master/api/API_VERSIONING.md#backwards-compatibility. Can you change the Java outer classname?

Reply:
Oh no. Yeah I'll do that instead.

Reply:
@spenceral you'll run into the issue that these are auto-generated by the protoformat tool based on the filename. You'll need to maybe update that with a special case.